alert.html

```diff
@@ -0,0 +1,43 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../vui-colors/colors.html">
+
```

Comment:
Yeah, I know Jeff has iterated on the design for this stuff, so I suspect when he has a few minutes to spare he'll be able to point to the latest stuff.